Keas.Mvc/Views/Report/Index.cshtml

```diff
@@ -63,5 +63,18 @@
     <div class="card-content">
         <a asp-action="PersonActions" class="btn btn-link">Actions taken against people in team <i class="fas fa-chevron-right fa-sm" aria-hidden="true"></i></a>
     </div>
-</div>
+</div>
+ @if ((await AuthorizationService.AuthorizeAsync(User, AccessCodes.Codes.SpaceMasterAccess)).Succeeded)
```

Comment:
this is ok but we should come up with a better way of authorizing reports, or cache this.  Currently the report page is going to be doing 2 queries per report so we are probably up to 10 already, just to figure out what the user can access.  One idea is to query for the user's roles server side and then use that to check if user isInRole inside the page.  Something to keep in mind anyway